src/GAP/iso_oscar_gap.jl

```diff
@@ -495,7 +495,7 @@ end
 #
 # Multivariate polynomial rings
 #
-@attributes AbstractAlgebra.Generic.MPolyRing # TODO: port this to AA
+@attributes AbstractAlgebra.Generic.MPolyRing # TODO: move this to AA
```

Comment:
added to AA in https://github.com/Nemocas/AbstractAlgebra.jl/pull/1494 but that has not yet been released